pkg/api-server/dataplane_overview_ws_test.go

```diff
@@ -97,6 +102,11 @@ var _ = Describe("Dataplane Overview WS", func() {
 	"dataplane": {
 		"networking": {
 			"address": "127.0.0.1",
+			"gateway": {
+                "tags": {
```

Comment:
formatting is off here. Did you use spaces instead of tabs?

Reply:
what is the default formatter we use for go - it is prettier?

Reply:
gofmt, but it does not format the content of strings, so for yamls and son we have to do it manually.